odoo/addons/base/tests/test_cli.py

```diff
@@ -117,12 +117,12 @@ def test_shell(self):
                 'print(message)\n'
                 'exit()\n'
             )
-        shell.wait()
+        self.assertFalse(shell.wait())
```

Comment:
I would rather we do `self.assertEqual(shell.wait(), 0, "exited with a non 0 code")`